parsl/executors/high_throughput/executor.py

```diff
@@ -627,46 +627,40 @@ def scale_in(self, blocks=None, block_ids=[], force=True, max_idletime=None):
              A time to indicate how long a block can be idle.
              Used along with force = False to kill blocks that have been idle for that long.
 
-        block_ids : list
-             List of specific block ids to terminate. Optional
-
         Returns
         -------
         List of job_ids marked for termination
         """
-        logger.debug(f"Scale in called, blocks={blocks}, block_ids={block_ids}")
-        if block_ids:
-            block_ids_to_kill = block_ids
+        logger.debug(f"Scale in called, blocks={blocks}")
+        managers = self.connected_managers()
+        block_info = {}  # block id -> list( tasks, idle duration )
+        for manager in managers:
```

Comment:
Not a change from the original code, but perhaps formalize the typing of `block_info`?  Again, "while here"?